shell/client/styleguide.html

```diff
@@ -1,6 +1,95 @@
 <template name="styleguide">
 {{!-- Insert whatever HTML you want to demonstrate here. --}}
-<h1>Sandstorm Style Guide</h1>
+<div class="styleguide">
+<h1>Sandstorm style guide</h1>
 
-<p>Work in progress.</p>
+<p>(note: this is a work in progress.)</p>
+
+<h2>Copy rules</h2>
+<ul>
+  <li>Be terse but clear.  Avoid technical terms where less-technical ones will do.</li>
+  <li>
+    Capitalize the first letter of the first word of a section heading or button.  Do not
+    capitalize the second word, or anything else, unless referring to a proper noun like Google or
+    GitHub.
+  </li>
+  <li>Use verbs for text on buttons. The button text should describe what clicking that button will do.</li>
+  <li>Use nouns for text which is a link.  The link text should indicate what the link will take you to.</li>
+  <li>
+    Use nouns for basically everything else, too, to communicate the information architecture.
+    Section headings should be nouns which describe the content of the section.  Labels should be
+    nouns which describe the content of the field.
+  </li>
+</ul>
+
+<h2>Style rules</h2>
+<ul>
+  <li>
+    Stick to a limited color palette.  (TODO: add specific colors)
+  </li>
+  <li>
+    Use typefaces no smaller than 13px for text content.  Readability is important and not everyone
+    has young eyes.
+  </li>
+  <li>
+    Make click targets no smaller than 32px square or 26px tall if appreciably wider.  Remember
+    that due to Fitts' Law, larger click targets are easier to interact with.
+  </li>
+</ul>
+
+<h2>Forms</h2>
+
+<ul>
+  <li>Use the <code>standard-form</code> class where appropriate.</li>
+  <li>
+    The primary action button of a form should be at the bottom right of the form. Secondary
+    buttons should line up to the left of that primary action button.
```

Comment:
I believe we have primary, secondary, and tertiary buttons. I'll try to find an example in our existing UI.